mix.exs

```diff
@@ -34,6 +34,8 @@ defmodule Ecto.Mixfile do
 
   defp deps do
     [{:poolboy, "~> 1.4"},
+     {:sbroker, "~> 0.6", github: "fishcakez/sbroker", branch: "gen"},
```

Comment:
It is for any connection process and nothing to do with `sbroker`. The mongo driver uses it for example https://github.com/ericmj/mongodb/blob/08b63a1aaca3bbf8f8f48daa90bc83630d46455f/lib/mongo/connection.ex#L10. @whatyouhide did a nice blog post about using it: http://andrealeopardi.com/posts/handling-tcp-connections-in-elixir/.

I am keen to remove from here because it would be an optional dep and the would be confusing. Also there could be a version mismatch with a driver.